README.md

```diff
@@ -305,6 +306,88 @@ RESET_PASSWORD_SECRET_KEY: The secret key for reset password functionality.
 VERIFICATION_SECRET_KEY: The secret key for email or user verification.
 ```
 
+## Production Deployment
+
+### Overview
+
+Deployment is streamlined through **Vercel**, with dedicated buttons for the **Frontend** and **Backend** applications. Both require specific configurations during and after deployment to ensure proper functionality.
+
+---
+
+### Frontend Deployment
+
+[![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/clone?repository-url=https%3A%2F%2Fgithub.com%2Fvintasoftware%2Fnextjs-fastapi-template%2Ftree%2Fmain%2Fnextjs-frontend&env=API_BASE_URL&envDescription=The%20API_BASE_URL%20is%20the%20backend%20URL%20where%20the%20frontend%20sends%20requests.)
+
+1. **Deploying the Frontend**  
+   - Click the **Frontend** button above to start the deployment process.  
+   - During deployment, you will be prompted to set the `API_BASE_URL`. Use a placeholder value (e.g., `http://`) for now, as this will be updated with the backend URL later.  
```

Comment:
http:// -> https://

Better ensure even the placeholder is https, to avoid missing the S on this later on.